### DISCUSSION TITLE
lost css properties in webpack build

### DISCUSSION THREAD

**Discussion by katzila** [cssinjs/jss#1617] [Q&A]
Hi! I am implementing a component library for React Typescript.
But there was a problem with webpack build.
When i installed own library with npm and tried to use any jss styled component - there is no css properties, only empty generated classes

Library repo: https://github.com/katzila/test-liba
npm: https://www.npmjs.com/package/oldman-test-lib

**Comment by fedeghe** [comment]
I have exactly the same problem [here](https://github.com/fedeghe/react-vgrid) and [there](https://github.com/fedeghe/react-vtable).
In dev mode everything is fine but when I build and import the component elsewhere some classes are empty
